apps/landing/src/styles/fonts.css

```
@@ -0,0 +1,18 @@
+/* Shared by the Tailwind pages (global.css) and the Starlight docs
+   (starlight.css), which do not load each other's stylesheets. */
+
+@font-face {
+  font-family: "InterVariable";
```

Stylelint (17.14.0)

[error] Expected no quotes around "InterVariable" (font-family-name-quotes)

(font-family-name-quotes)